scipy/fft/tests/test_basic.py

```diff
@@ -327,6 +303,7 @@ def test_all_1d_norm_preserving(self, xp):
                     tmp = back(tmp, n=n, norm=norm)
                     xp_assert_close(xp_test.linalg.vector_norm(tmp), x_norm)
 
+    @skip_if_array_api(np_only=True)
```

Comment:
I think I may have noticed that in some cases you also added the `xp` conversion to the test for NumPy only. I suppose it doesn't matter much since the marker will remind us either way.

Reply:
In order to use `pytestmark=array_api_compatible` every test needs to have an `xp` parameter, even those which are `np_only`. I'm not sure how we'll want to resolve this long term, but in any case a big refactor will be needed once array API behaviour stops being hidden behind an environment variable, and I think this is at least "ok" until then.